### PR TITLE
Added backup and restore topics

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/backup-and-restore/

Preview: 
- https://openshift-docs-git-backup-restore-gnelson.vercel.app/openshift-acs/master/backup_and_restore/backing-up-acs.html
- https://openshift-docs-git-backup-restore-gnelson.vercel.app/openshift-acs/master/backup_and_restore/restore-acs.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer PTAL

File: modules/on-demand-backups-roxctl-admin-pass.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * dir/filename.adoc
```
Comment (gaurav-nelson) [inline modules/on-demand-backups-roxctl-admin-pass.adoc:3]: Note to self: Fix assembly paths in all modules.

File: backup_and_restore/backing-up-acs.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+toc::[]
+
+[role="_abstract"]
+You can perform data backups for {product-title}.
+You can use these backups for data restoration in the case of an infrastructure disaster, or corrupt data.
+
+You can configure automatic or on-demand backups by integrating with Amazon S3 or Google Cloud Storage.
+Or you can perform on-demand backups by using the `roxctl` command-line interface (CLI).
+
+The backup includes {product-title} entire database, which includes all configurations, resources, events, and certificates.
```
Comment (bergerhoffer) [inline backup_and_restore/backing-up-acs.adoc:15]: I think there needs to be a "the" in there somewhere. Maybe:

```
The backup includes the entire {product-title} database,
```

File: modules/on-demand-backups-roxctl-admin-pass.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/backing-up-acs.adoc
+:_module-type: PROCEDURE
+[id="on-demand-backups-roxctl-admin-pass_{context}"]
+= On-demand backups by using the administrator password
+
+[role="_abstract"]
+You can backup the entire database of {product-title} by using your administrator password.
```
Comment (bergerhoffer) [inline modules/on-demand-backups-roxctl-admin-pass.adoc:9]: You can backup -> You can back up

File: modules/on-demand-backups-roxctl-admin-pass.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+[source,terminal]
+----
+$ export CENTRAL_ADDRESS=<address>:<port_number>
+----
+. Run the `backup` command:
+* For {product-title} version 3.0.55 or later:
++
+[source,terminal]
+----
+$ roxctl -p <admin-password> -e "$CENTRAL_ADDRESS" central backup
```
Comment (bergerhoffer) [inline modules/on-demand-backups-roxctl-admin-pass.adoc:29]: \<admin-password> -> <admin_password>

File: modules/on-demand-backups-roxctl-admin-pass.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+----
+* For {product-title} version 3.0.54 or older:
++
+[source,terminal]
+----
+$ roxctl -p <admin-password> -e "$CENTRAL_ADDRESS" central db backup
```
Comment (bergerhoffer) [inline modules/on-demand-backups-roxctl-admin-pass.adoc:35]: \<admin-password> -> <admin_password>

File: modules/on-demand-backups-roxctl-api.adoc [hunk 6]
```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/backing-up-acs.adoc
+:_module-type: PROCEDURE
+[id="on-demand-backups-roxctl-api_{context}"]
+= On-demand backups by using an API token
+
+[role="_abstract"]
+You can backup the entire database of {product-title} by using an API token.
```
Comment (bergerhoffer) [inline modules/on-demand-backups-roxctl-api.adoc:9]: You can backup -> You can back up

File: modules/restore-acs-roxctl-admin-pass.adoc [hunk 7]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/restore-acs.adoc
+:_module-type: PROCEDURE
+[id="restore-acs-roxctl-admin-pass_{context}"]
+= Restore by using the administrator password
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-admin-pass.adoc:6]: Restore by -> Restoring by

File: modules/restore-acs-roxctl-api.adoc [hunk 8]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/restore-acs.adoc
+:_module-type: PROCEDURE
+[id="restore-acs-roxctl-api_{context}"]
+= Restore by using an API token
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-api.adoc:6]: Restore by -> Restoring by

File: modules/restore-acs-roxctl-admin-pass.adoc [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+[role="_abstract"]
+You can backup the entire database of {product-title} by using your administrator password.
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-admin-pass.adoc:9]: Should this be "You can restore" ?

File: modules/restore-acs-roxctl-admin-pass.adoc [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+.Prerequisites
+
+* You must have {product-title} backup file.
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-admin-pass.adoc:13]: must have -> must have a

File: modules/restore-acs-roxctl-admin-pass.adoc [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
++
+[source,terminal]
+----
+$ export CENTRAL_ADDRESS=<address>:<port_number>
+----
+. Run the `restore` command:
++
+[source,terminal]
+----
+$ roxctl -p <admin-password> -e "$CENTRAL_ADDRESS" central db restore <backup_file>
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-admin-pass.adoc:28]: \<admin-password> -> <admin_password>

File: modules/restore-acs-roxctl-api.adoc [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+[role="_abstract"]
+You can restore the entire database of {product-title} by using an API token.
+
+.Prerequisites
+
+* You must have {product-title} backup file.
```
Comment (bergerhoffer) [inline modules/restore-acs-roxctl-api.adoc:13]: must have -> must have a

**Review by bergerhoffer — COMMENTED** [review]
LGTM, merging!

**Pull request merged**